### ISSUE TITLE
Go back to previous modal when clicked, instead of Close

### ISSUE THREAD

**Issue by umangutkarsh** [RocketChat/Apps.Github22#147]
## Why do we need this ?
To make the user experience better.
Currently, when this command -> `/github  Username/RepositoryName pulls` is entered, action buttons along with link to PR are rendered. And in the PR actions modal when the desired buttons are clicked, for example - View File, View Changes, Comments, etc, the desired modal is opened up but then there is no option to go back, instead the user has to close the modal and again re-open it to select other option in the PR actions modal.

As a GitHub App developer

**I need to**: Implement a feature to go back to previous modal (PR actions modal) instead of closing it.

**So That**: The user doesn't have to re-open the modal again and again which doesn't provide good user experience.

## Acceptance Criteria

- [ ] Integrate `Go Back` option instead of `Close` submit button in PR Actions modal.

## Screenshots/Record
[gh-ux.webm](https://github.com/RocketChat/Apps.Github22/assets/95426993/1b48ca57-b529-4fa7-a324-eafcc7982b8a)

**Comment by umangutkarsh** [comment]
Hi @samad-yar-khan . What do you think about this? Please let me know. Thanks.

**Comment by samad-yar-khan** [comment]
@umangutkarsh makes sense. I believe this was earlier inbuilt in RocketChat and they then removed this stacking. I think stacking its like an application UI would make for a better UX. Would you like to work on this ?

**Comment by umangutkarsh** [comment]
Yes sure @samad-yar-khan . Is it okay if I give you updates after I make some progress?

Thanks.

**Comment by samad-yar-khan** [comment]
yes, please feel free to join the channel : https://open.rocket.chat/channel/GitHub-App-Community-Developers

**Comment by devanshkansagra** [comment]
Hey @samad-yar-khan and @umangutkarsh, this issue has been idle for a while. If it's okay with both of you, can I work on this issue/feature?

**Comment by devanshkansagra** [comment]
Hey @samad-yar-khan and @umangutkarsh, a suggestion, can we add a normal action button go-back at the end? And in that on clicking it, it will display the modal which was displayed when `/github  Username/RepositoryName pulls` is executed. I am approaching in this way because I have seen that there is not so much deep nesting of modals, we have upto level 2, meaning suppose when clicking on `view file` action button the view file modal will show up and next there is no modal. 

There may still be some minor inconsistencies in the ux, such as the need to scroll all the way down to navigate back. However, this small inconsistency is not as significant as having to execute the entire command and re-open the modal again. 

So what are your thoughts on this approach?
